app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -109,6 +118,27 @@ def parse_build_pod(build_pod)
                                                            build_pod.status.config.try(:name))
         new_result
       end
+
+      def parse_template_parameters(parameters)
```

Comment:
@zakiva I see in other parse functions we use symbols not strings, why we use strings over here?